### PR TITLE
Oldpr

### PR DESCRIPTION
**Pull request by CedricCabessa**

### CODE REVIEW

File: gm_pr/models.py [hunk 1]
```diff
@@ -2,7 +2,8 @@ class Pr:
     """ Stupid class wrapper for pr properties
     """
     def __init__(self, url="", title="", updated_at="", user="",
-                 repo="", nbreview=0, plusone=0, lgtm=0, milestone=None, label=None):
+                 repo="", nbreview=0, plusone=0, lgtm=0, milestone=None, labels=None,
+                 isOld=False):
```
Comment (srats) [inline gm_pr/models.py]: python style: ```is_old```

File: gm_pr/PrFetcher.py [hunk 2]
```diff
@@ -24,7 +26,23 @@ def fetch_data(project_name, url, org):
...
+                                   'color' : lbl['color'],
+                               })
+
+            date = datetime.strptime(detail_json['updated_at'], '%Y-%m-%dT%H:%M:%SZ')
+            isOld = False
+            if (now - date).days >= settings.OLD_PERIOD:
+                if not labels and None in settings.OLD_LABEL:
+                    isOld = True
+                else:
+                    for lbl in labels:
```
Comment (srats) [inline gm_pr/PrFetcher.py]: ```isOld = settings.OLD_LABEL in [ label['name'] for label in labels]```

Comment (srats) [inline gm_pr/PrFetcher.py]: Or, at least, add a ```break``` after ```isOld = True``` (line 44)

File: gm_pr/settings.py [hunk 3]
```diff
@@ -174,6 +174,12 @@
 
 TOP_LEVEL_URL = "https://api.github.com"
 ORG = "Genymobile"
+# Number of days a PRs without update can be flagged as OLD.
+OLD_PERIOD=4
+# only PRs with one of those labels can be considered as OLD.
+# Use 'None' for "no label"
```
Comment (srats) [inline gm_pr/settings.py]: ```#Use None for "no label"``` (without the ```'```) ?

**Comment by CedricCabessa** [comment]
Merci @srats , vous êtes très pythonique

**Comment by srats** [comment]
Moutonne-y donc ! :sheep: :+1:

**Pull request merged**